.travis.yml

```diff
@@ -45,18 +54,21 @@ before_install: |
     pyenv install $PYTHON 
     pyenv virtualenv $PYTHON MYVERSION
     source ~/.pyenv/versions/MYVERSION/bin/activate
+    pyenv global $PYTHON
     python --version 
     python -c 'import sys;print(sys.version)' 
     python -c 'import sys;print(sys.executable)' 
   fi
 install:
 # we have this here so we can see where python is installed and hardcode in our tests
```

Comment:
Hardcode how? Can you just pass it in through an environment variable or command-line option? E.g.
```shell
PYTHON_EXE=`which python`
```

Reply:
This was for OSX tests. I need this to be the global setting so the unit tests can pick the standard python path. I could introduce this env variable as well.